source/common/upstream/upstream_impl.h

```diff
@@ -40,6 +40,8 @@
 #include "common/upstream/outlier_detection_impl.h"
 #include "common/upstream/resource_manager_impl.h"
 
+#include "server/init_manager_impl.h"
```

Comment:
Please move init_manager_impl out of server/ and into a new subdirectory in source/ called init/. Feel free to do this in a followup but please add a TODO.

Reply:
Added TODO in init_manager_impl.h. Thanks.